packages/api/tests/lambdas/sf-event-sqs-to-db-records/test-index.js

```diff
@@ -0,0 +1,357 @@
+  t.context.collectionId = constructCollectionId(
+    t.context.collection.name,
+    t.context.collection.version
+  );
+
+  const stateMachineName = cryptoRandomString({ length: 5 });
+  t.context.stateMachineArn = `arn:aws:states:${fixture.region}:${fixture.account}:stateMachine:${stateMachineName}`;
+
+  t.context.executionName = cryptoRandomString({ length: 5 });
+  t.context.executionArn = `arn:aws:states:${fixture.region}:${fixture.account}:execution:${stateMachineName}:${t.context.executionName}`;
+
+  t.context.parentExecutionArn = `machine:${cryptoRandomString({ length: 5 })}`;
+  t.context.asyncOperationId = uuidv4();
+
+  t.context.provider = {
+    id: `provider${cryptoRandomString({ length: 5 })}`,
+    host: 'test-bucket',
+    protocol: 's3',
+  };
+
+  t.context.pdrName = cryptoRandomString({ length: 10 });
+  t.context.pdr = {
+    name: t.context.pdrName,
+    PANSent: false,
+    PANmessage: 'test',
+  };
+
+  t.context.granuleId = cryptoRandomString({ length: 10 });
+  const files = [fakeFileFactory()];
+  const granule = fakeGranuleFactoryV2({ files, granuleId: t.context.granuleId });
+
+  t.context.cumulusMessage = {
+    cumulus_meta: {
+      workflow_start_time: 122,
+      cumulus_version: t.context.postRDSDeploymentVersion,
+      state_machine: t.context.stateMachineArn,
+      execution_name: t.context.executionName,
+      parentExecutionArn: t.context.parentExecutionArn,
+      asyncOperationId: t.context.asyncOperationId,
+    },
+    meta: {
+      status: 'running',
+      collection: toCamel(t.context.collection),
+      provider: t.context.provider,
+    },
+    payload: {
+      key: 'my-payload',
+      pdr: t.context.pdr,
+      granules: [granule],
+    },
+  };
+
+  const collectionResponse = await t.context.knex(tableNames.collections)
+    .insert(t.context.collection)
+    .returning('cumulus_id');
+  t.context.collectionCumulusId = collectionResponse[0];
+
+  const providerResponse = await t.context.knex(tableNames.providers)
+    .insert({
+      name: t.context.provider.id,
+      host: t.context.provider.host,
+      protocol: t.context.provider.protocol,
+    })
+    .returning('cumulus_id');
+  t.context.providerCumulusId = providerResponse[0];
+
+  t.context.shouldWriteExecutionStub = shouldWriteExecutionStub;
+  t.context.shouldWriteExecutionStub.resolves(true);
+  t.context.shouldWriteExecutionStub.resetHistory();
+});
+
+test.after.always(async (t) => {
+  const {
+    executionModel,
+    pdrModel,
+    granuleModel,
+  } = t.context;
+  await executionModel.deleteTable();
+  await pdrModel.deleteTable();
+  await granuleModel.deleteTable();
+  sandbox.restore();
+  await t.context.knex.destroy();
+  await t.context.knexAdmin.raw(`drop database if exists "${t.context.testDbName}"`);
+  await t.context.knexAdmin.destroy();
+});
+
+test.serial('writeRecords() only writes records to Dynamo if shouldWriteExecutionToRDS returns false', async (t) => {
```

Comment:
This test text changed - given the nature of a re-org PR, were there any other non-cosmetic changes?